### DISCUSSION TITLE
Coin not reach to Binance address

### DISCUSSION THREAD

**Discussion by Naymyokyaw2024** [blockscout/blockscout#10105] [General]
I have wrongly send the coin to Binance. Kindly help.1.6 FTN.
From -0x4e3185673e7aa44bff1e686375690248825126ad

To Binance
0xccaebd3a655d6325a61c126a5aaf8d1886837519

![Screenshot_2024-05-21-20-43-51-087_com fastex wallet dev](https://github.com/blockscout/blockscout/assets/170441843/832b3a85-730f-4a19-a64d-955c3f0deab4)

**Comment by sl1depengwyn** [comment]
This repo is for Blockscout-related questions. Blockscout is an open source blockchain explorer. Thus, your question is out of scope, where the Blockscout team can help, you better contact the chain support directly.
